code/game/objects/items/devices/radio/headset.dm

```diff
@@ -231,7 +236,8 @@
 			if(user.mind && user.assigned_squad && user.hud_used && user.hud_used.locate_leader)
 				user.hud_used.locate_leader.alpha = 255
 				user.hud_used.locate_leader.mouse_opacity = 1
-
+	if(camera)
+		camera.c_tag = user.name
 	..()
```

Comment:
I know it's not your code but if you're touching the proc you could clean it up.
```suggestion
	return ..()
```